utils.js

```diff
@@ -6860,7 +6874,10 @@ export const extractJarArchive = async function (
               technique = "hash-comparison";
             }
           } catch (err) {
-            console.log(err);
+            if (err && err.message && !err.message.includes("404")) {
```

Comment:
Can we use the err.code to do this?

https://github.com/sindresorhus/got/blob/main/documentation/8-errors.md